react/features/base/participants/preloadImage.native.ts

```diff
@@ -9,14 +6,16 @@ import { isIconUrl } from './functions';
  * Tries to preload an image.
  *
  * @param {string | Object} src - Source of the avatar.
+ * @param {boolean} _isUsingCORS - Used on web.
  * @returns {Promise}
  */
-export function preloadImage(src: string | Object): Promise<Object> {
+export function preloadImage(src: string | Object, _isUsingCORS: boolean): Promise<any> {
```

Comment:
Seems that the web function has a `useCors`s and `tryOnce` param, and both native and web have the same payload for the promise resolve
```
{
  src: string,
  isUsingCORS: boolean
}
```

Reply:
Not really. The native also has a case of `Promise.resolve(src)`. So then the payload can be either like you wrote or just `string | Object`.